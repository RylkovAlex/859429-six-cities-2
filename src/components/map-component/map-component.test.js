import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import MapComponent from './map-component';

Enzyme.configure({adapter: new Adapter()});

it(`MapComponent renders after relaunch`, () => {
  const props = {
    config: {
      center: [0, 0],
      zoom: 0,
      points: [
        {
          position: [0, 0],
          id: 0,
        }
      ]
    },
    activePointId: 0,
  };

  const tree = shallow(<MapComponent {...props}/>);
  expect(toJson(tree)).toMatchSnapshot();

});
