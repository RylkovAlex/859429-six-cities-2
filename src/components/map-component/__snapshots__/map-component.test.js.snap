// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapComponent renders after relaunch 1`] = `
<Map
  center={
    Array [
      0,
      0,
    ]
  }
  style={
    Object {
      "height": "100%",
    }
  }
  zoom={0}
>
  <ForwardRef(Leaflet(LayersControl))
    position="topright"
  >
    <BaseLayer
      checked={true}
      name="BaseMap"
    >
      <ForwardRef(Leaflet(TileLayer))
        attribution="&copy; <a href=\\"https://www.openstreetmap.org/copyright\\">OpenStreetMap</a> contributors &copy; <a href=\\"https://carto.com/attributions\\">CARTO</a>"
        url="https://{s}.basemaps.cartocdn.com/rastertiles/voyager/{z}/{x}/{y}{r}.png"
      />
    </BaseLayer>
    <BaseLayer
      name="HyddaMap"
    >
      <ForwardRef(Leaflet(TileLayer))
        attribution="Tiles courtesy of <a href=\\"http://openstreetmap.se/\\" target=\\"_blank\\">OpenStreetMap Sweden</a> &mdash; Map data &copy; <a href=\\"https://www.openstreetmap.org/copyright\\">OpenStreetMap</a> contributors"
        url="https://{s}.tile.openstreetmap.se/hydda/full/{z}/{x}/{y}.png"
      />
    </BaseLayer>
  </ForwardRef(Leaflet(LayersControl))>
  <ForwardRef(Leaflet(Marker))
    icon={
      NewClass {
        "_initHooksCalled": true,
        "options": Object {
          "iconUrl": "/img/pin-active.svg",
        },
      }
    }
    key="0"
    onClick={[Function]}
    position={
      Array [
        0,
        0,
      ]
    }
  />
</Map>
`;
